lib/widgets/scaffold.dart

```diff
@@ -163,4 +167,58 @@ class AppScaffold extends StatelessWidget {
       ),
     );
   }
+
+  ///
+  /// Build content body based on is web version.
+  /// When web version ad an action bar to right side.
```

Comment:
This doesn't really make sense. Maybe "Build content body based on platform. On web, there are action buttons on the right side."?

Reply:
Changed comments:
https://github.com/student-hub/acs-upb-mobile/pull/250/commits/8c0e90bd1db807c70ea5422527345d766711da4c#diff-91af2d9745e86244940959ec39c99cca9b8b43c1502042d85f16fcc6a311030bR135-R137